### PR TITLE
fix(release-notes): adapt release-note related Gaja-X

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR adapts the release notes related to the Standards and adds the suggestion provided by (@jjeroch and @mihahnberlin) . 

image

### CODE REVIEW

File: src/pages/release-notes.md [hunk 1]
```diff
@@ -63,12 +63,13 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
...
 
-- This suspension only affects the [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) role within the following CACs:
-  - [CX-0001](/docs/standards/CX-0001-EDCDiscoveryAPI#15-proof-of-conformity)
-    - The provided SD documents MUST be GAIA-X compliant, i.e. MUST provide a compliance credential issued from GAIA-X AISBL. – to be temporarily deactivated
-  - [CX-0006](/docs/standards/CX-0006-RegistrationAndInitialOnboarding#238-gaia-x).
-    - Onboarding service providers / core service providers MUST ensure the creation of a Self-Description (SD) of the Legal Person as part of the application verification process. The SD MUST be stored in a central location and MUST be in line with the GAIA-X standards/guidelines
+Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [Onboarding service providers](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles within the following CACs:
+
+- [CX-0001](/docs/standards/CX-0001-EDCDiscoveryAPI#15-proof-of-conformity)
```
Comment (jSchuetz88) [inline src/pages/release-notes.md]:
```suggestion
```

File: src/pages/release-notes.md [hunk 2]
```diff
@@ -63,12 +63,13 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
...
 
-- This suspension only affects the [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) role within the following CACs:
-  - [CX-0001](/docs/standards/CX-0001-EDCDiscoveryAPI#15-proof-of-conformity)
-    - The provided SD documents MUST be GAIA-X compliant, i.e. MUST provide a compliance credential issued from GAIA-X AISBL. – to be temporarily deactivated
-  - [CX-0006](/docs/standards/CX-0006-RegistrationAndInitialOnboarding#238-gaia-x).
-    - Onboarding service providers / core service providers MUST ensure the creation of a Self-Description (SD) of the Legal Person as part of the application verification process. The SD MUST be stored in a central location and MUST be in line with the GAIA-X standards/guidelines
+Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [Onboarding service providers](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles within the following CACs:
```
Comment (jSchuetz88) [inline src/pages/release-notes.md]:
```suggestion
Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [OSP](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles:
```

**Review by jSchuetz88 — COMMENTED** [review]
Looks good to me, just two minor suggestions.

File: src/pages/release-notes.md [hunk 3]
```diff
@@ -63,12 +63,12 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
...
 
-- This suspension only affects the [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) role within the following CACs:
-  - [CX-0001](/docs/standards/CX-0001-EDCDiscoveryAPI#15-proof-of-conformity)
-    - The provided SD documents MUST be GAIA-X compliant, i.e. MUST provide a compliance credential issued from GAIA-X AISBL. – to be temporarily deactivated
-  - [CX-0006](/docs/standards/CX-0006-RegistrationAndInitialOnboarding#238-gaia-x).
-    - Onboarding service providers / core service providers MUST ensure the creation of a Self-Description (SD) of the Legal Person as part of the application verification process. The SD MUST be stored in a central location and MUST be in line with the GAIA-X standards/guidelines
+Impact: [CSP-B](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#core-service-provider-b) & [OSP](/docs/operating-model/who-roles-in-the-catena-x-ecosystem#onboarding-service-provider) roles:
+
+  - The provided SD documents MUST be GAIA-X compliant, i.e. MUST provide a compliance credential issued from GAIA-X AISBL. – to be temporarily deactivated
```
Comment (jSchuetz88) [inline src/pages/release-notes.md]:
```suggestion
```

**Review by jSchuetz88 — APPROVED** [review]
As discussed and agreed. Looks good to me.

**Comment by mihahnberlin** [comment]
lgtm :-)

**Pull request merged**